### ISSUE TITLE
Unable to set breakpoints in code-server

### ISSUE THREAD

**Issue by maloo** [WebFreak001/code-debug#292]
If submitting a bug please make sure

- [X] If you are using gdb
  - [X] `gdb --version` >= 7.7.1 (GNU gdb (Ubuntu 9.2-0ubuntu1~20.04) 9.2)
  - [X] it works on the command line with `gdb`
  - [X] `cwd` and `target` are properly set

Neither F9 or click on line allow setting breakpoint when running inside code-server. Other debugger plugins don't have this issue, so it seems to be a Native Debug issue.

To reproduce you can use the following docker setup:

Dockerfile:
```
FROM ubuntu:20.04

EXPOSE 8888

ENV DEBIAN_FRONTEND=noninteractive
RUN apt update &&\
    apt install -y \
    build-essential \
    vim \
    wget \
    curl \
    gdb \
    lldb \
    && rm -rf /var/lib/apt/lists/*

RUN curl -fsSL https://code-server.dev/install.sh | sh

RUN code-server --install-extension webfreak.debug

CMD code-server --host=0.0.0.0 --port=8888 --auth none
```

docker-compose-yml:
```
version: "3.9"
   
services:
  code-server:
    image: code-server:latest
    ports:
    - 8888:8888
    volumes:
    - ./data:/data
```

./data/main.cpp:
```
#include <stdio.h>

int main()
{
    printf("Hello World!\n");

    return 0;
}
```

Build: `docker build -t code-server:latest .`

Start: `docker compose up`

Test: Open localhost:8888 in a browser and try setting a breakpoint in a .cpp file.

**Comment by brownts** [comment]
Hi @maloo, I believe this has been addressed by #280, so should be in the next release of this extension.  However, in the meantime, you should be able to work around this by adjusting your setting in VSCode to allow setting breakpoints anywhere:
- File > Preferences > Settings > Features > Debug
  - Enable the "Allow Breakpoints Everywhere" setting

**Comment by GitMensch** [comment]
Should be fixed, so closing. @maloo Could you please recheck (I don't have a docker setup running)? If there are still issues we can reopen and analyze further.
